themes/hugo-steam-theme/layouts/partials/footer.html

```diff
@@ -7,9 +9,18 @@
 		    {{ with .Site.Params.github }}
 		    <a href="//github.com/{{ . }}" target="_blank" title="GitHub"><i class="fa fa-2x fa-fw fa-github"></i> <span class="hidden">GitHub</span></a>&nbsp;
 		    {{ end }}
+		    {{ with .Site.Params.medium }}
+		    <a href="//medium.com/{{ . }}" target="_blank" title="Medium"><i class="fa fa-2x fa-fw fa-medium"></i> <span class="hidden">Medium</span></a>&nbsp;
+		    {{ end }}
+		    {{ with .Site.Params.angellist }}
+		    <a href="//angel.co/{{ . }}" target="_blank" title="Angel List"><i class="fa fa-2x fa-fw fa-angellist"></i> <span class="hidden">AngelList</span></a>&nbsp;
+		    {{ end }}
+		    {{ with .Site.Params.linkedin }}
+		    <a href="//linkedin.com/company-beta/{{ . }}" target="_blank" title="LinkedIn"><i class="fa fa-2x fa-fw fa-linkedin"></i> <span class="hidden">LinkedIn</span></a>&nbsp;
```

Comment:
Can you change `hidden` here for `sr-only`. See impl here: https://github.com/twbs/bootstrap/blob/2085c142679563189dbc34ec6bf4b00e32b8f615/scss/mixins/_screen-reader.scss#L6-L16